README.md

```diff
@@ -60,6 +60,7 @@ Upload Lambda to AWS and _star_ this repository if it works as expected!!
 | index  | The name of ElasticSearch index (string). If not provided will set the same as DynamoDB table name | optional
 | type  | The type of the ElasticSearch document (string). If not provided will set the same as DynamoDB table name | optional
 | refresh  | Force ElasticSearch refresh its index immediately [more here](https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-refresh.html) (boolean). Default: true | optional
+| transformFunction  | A function to transform each record before sending them to ES. Applies to INSERT and UPDATE operations | optional
```

Comment:
I would explain more deeply with the example

Reply:
Maybe improving the description would work. You can pass a function which return the record with the transformations. And then refer the user to the test files.